log/logging.go

```diff
@@ -241,6 +241,7 @@ const (
 	DefaultFormat   LogFormat = "" // Use env variable, defaulting to PlainTextFormat
 	PlainTextFormat LogFormat = "text"
 	JsonFormat      LogFormat = "json"
+	SumoFormat      LogFormat = "sumo"
```

Comment:
In the interest of not using vendor prefixes or names, let's rename this to `KeyValueFormat`